src/ripple/app/main/Application.cpp

```diff
@@ -1645,8 +1646,7 @@ bool ApplicationImp::setup()
                     return false;
                 }
 
-                if(invokeStart)
-                    handler->start();
+                handler->start();
```

Comment:
What if start fails?

Reply:
Well, the error will get logged in the start function and any exception would have also been caught in that method. Not sure we need to do anything else.

Reply:
On failure to grab a handler instance or to initialize it, we return false, why is start treated differently?

Reply:
Oh I misunderstood you. Done